brands/amenity/fast_food.json

```diff
@@ -865,10 +865,14 @@
     "tags": {
       "amenity": "fast_food",
       "brand": "Herfy",
```

Comment:
While we’re at it, there should be a separate entry for Saudi Arabia, the UAE, Kuwait, and Bahrain with `brand` and `name` set to `هرفي`.

Reply:
I’m new to localization tagging.  Is that because the `name:ar` will present the POI information in Arabic to Arabic users but the POI will get tagged with `name=Herfy` regardless?

Reply:
Right, and multilingual renderers like the one on osm.org will just show “Herfy”, despite the usual practice of using local languages in that tag. If you search Nominatim for “Herfy”, most existing results are tagged with Arabic in the `name` tag. These features would get “upgraded” to English.

Reply:
But do we know the multilingual branding is broken up by country?

I see some `name=herfy` POIs in Saudia Arabia too: https://overpass-turbo.eu/s/L0A

@waso99 do you have any insight into this?

Reply:
> Edit: I added a separate entry for هرفي with Saudi Arabia, the UAE, Kuwait, and Bahrain country codes.

Looks good to me, thanks!